.github/workflows/build.yml

```
@@ -81,7 +75,7 @@ jobs:
           curl --fail --request POST \
           --form token=${{ secrets.TOKEN }} \
           --form ref=main \
-          --form "variables[GITHUB_TAG]=${{ steps.get_version.outputs.VERSION }}" \
+          --form "variables[GITHUB_TAG]=$VERSION" \
```

Qui forse va così? Non ho provato, lo dico confrontando con gli step precedenti

```suggestion
          --form "variables[GITHUB_TAG]=${{ env.VERSION }}" \
```

---

credo sia equivalente qui, il problema secondo me c'era quando $VERSION aveva `:` attaccati... cmq corretto, li teniamo allineati.